### PR TITLE
INT-5072 configure logging

### PR DESCRIPTION
**Pull request by jflinchbaugh**
Configure debug logging and files, so support can find them.

JIRA: https://issues.sonatype.org/browse/INT-5072
Build: https://jenkins.ci.sonatype.dev/job/integrations/job/cloud/job/operator-iq/job/feature-snapshots/job/INT-5072-configure-logging/

### CODE REVIEW

**Comment by jflinchbaugh** [comment]
the same configs are introduced to 3 different files: 2 as yaml and 1 as json.

File: scripts/templates/values.yaml [hunk 1]
```diff
@@ -15,17 +15,67 @@ iq:
...
+      # Logger-specific settings.
+      loggers:
+        "com.sonatype.insight.scan": INFO
+        "eu.medsea.mimeutil.MimeUtil2": INFO
+        "org.apache.http": INFO
+        "org.apache.http.wire": ERROR
+        "org.eclipse.birt.report.engine.layout.pdf.font.FontConfigReader": WARN
+        "org.eclipse.jetty": INFO
+        "org.apache.shiro.web.filter.authc.BasicHttpAuthenticationFilter": INFO # WARNING: This reveals credentials at DEBUG level
+        "com.sonatype.insight.audit":
```
Comment (collinpeters) [inline scripts/templates/values.yaml:51]: One forward looking thought. These are from the IQ config.yml which is good, but eventually they will be stale. Any thoughts on how we grab from the main config.yml

**Review by collinpeters — APPROVED** [review]
+1

**Review by hectorhuol — APPROVED** [review]
+1
I am not too familiar with this operators-iq repo, but changes look good to me

File: scripts/templates/values.yaml (hunk 1 above)
Comment (jflinchbaugh) [reply scripts/templates/values.yaml:51]: noted: https://issues.sonatype.org/browse/INT-5938

**Pull request merged**